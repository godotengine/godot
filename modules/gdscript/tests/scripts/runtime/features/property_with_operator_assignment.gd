#GDTEST_OK
var prop: int = 0:
	get:
		return prop
	set(value):
		prop = value % 7

func test():
	for i in 7:
		prop += 1
		print(prop)
