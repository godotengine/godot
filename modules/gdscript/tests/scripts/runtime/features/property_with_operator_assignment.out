GDTEST_OK
1
2
3
4
5
6
0
